### ISSUE TITLE
Error when using no partials

### ISSUE THREAD

**Issue by tmcconechy** [stephanepericat/beardcomb#6]
This is great! Small issue is that you must supply partials values. I tried to use this because all partials are on disk:
```
res.render('controls/index', {
 locals: {
   enableLiveReload: app.locals.enableLiveReload
   }
   foundation: 'layout.html'
  });
```
without partials and got the error.

```
/Users/tmcconechy/Dev/controls/node_modules/beardcomb/lib/comb.js:35
		if(part && part[1] === '>' && !options.partials[part[2]]) {
		                                               ^
TypeError: Cannot read property 'header' of undefined
    at /Users/tmcconechy/Dev/controls/node_modules/beardcomb/lib/comb.js:35:50`
```

Workaround is to provide empty partials like so:
```
res.render('controls/index', {
        locals: {
          enableLiveReload: app.locals.enableLiveReload
        },
        partials: {},
        foundation: 'layout.html'
    });
```

**Comment by stephanepericat** [comment]
Hi,

sorry for getting back to you so late. This modules is up for an overhaul to work with express 4.
I will address the issues you reported in the process.

Ste

**Comment by stephanepericat** [comment]
fixed in 0.4.0
